app/models/sms/decoder.rb

```diff
@@ -53,9 +53,9 @@ def decode
 
         # look up the questioning object for the specified rank and store it in the @qing instance variable
         find_qing
-
-      # otherwise, we add the token to the value variable and proceed
-      else
+      elsif index == 0 # if this is the first token, raise error
+        raise_decoding_error("question_doesnt_exist", rank: tok)
```

Comment:
Should we perhaps add a more descriptive error message for this one? e.g. "Sorry, but '#{token}' is not a valid answer. Be sure to include a period after the question number."